src/node.zig

```diff
@@ -307,7 +308,105 @@ fn encode_napi_results_array(
+                );
+
+                try translate.u128_into_object(
+                    env,
+                    napi_object,
+                    "debit_account_id",
+                    result.debit_account_id,
+                    "Failed to set property \"debit_account_id\" of transfer lookup result.",
+                );
+
+                try translate.u128_into_object(
+                    env,
+                    napi_object,
+                    "credit_account_id",
+                    result.credit_account_id,
+                    "Failed to set property \"credit_account_id\" of transfer lookup result.",
+                );
+
+                try translate.u128_into_object(
+                    env,
+                    napi_object,
+                    "user_data",
+                    result.user_data,
+                    "Failed to set property \"user_data\" of transfer lookup result.",
+                );
+
+                try translate.byte_slice_into_object(
+                    env,
+                    napi_object,
+                    "reserved",
+                    &result.reserved,
+                    "Failed to set property \"reserved\" of transfer lookup result.",
+                );
+
+                try translate.u64_into_object(
+                    env,
+                    napi_object,
+                    "timeout",
+                    result.timeout,
+                    "Failed to set property \"timeout\" of transfer lookup result.",
+                );
+
+                try translate.u32_into_object(
+                    env,
+                    napi_object,
+                    "code",
+                    @intCast(u32, result.code),
+                    "Failed to set property \"code\" of transfer lookup result.",
+                );
+
+                try translate.u32_into_object(
+                    env,
+                    napi_object,
+                    "flags",
+                    @bitCast(u32, result.flags),
+                    "Failed to set property \"flags\" of transfer lookup result.",
+                );
+
+                try translate.u64_into_object(
+                    env,
+                    napi_object,
+                    "amount",
+                    result.amount,
+                    "Failed to set property \"amount\" of transfer lookup result.",
+                );
+
+                try translate.u64_into_object(
+                    env,
+                    napi_object,
+                    "timestamp",
+                    result.timestamp,
+                    "Failed to set property \"timestamp\" of transfer lookup result.",
+                );
+
+                try translate.set_array_element(
+                    env,
+                    napi_array,
+                    i,
+                    napi_object,
+                    "Failed to set element in results array.",
+                );
+            }
+        },
+        Account => {
```

Comment:
Perhaps we should just move this `Account` branch prong ahead of the `Transfer` prong, so it's alphabetical order. We typically always have the account stuff ahead of the transfer stuff (e.g. in the `tigerbeetle.zig` types file or in the state machine).

Reply:
I agree. Done this. :-)